content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -8,27 +8,33 @@ weight: 40
 ---
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+{{< feature-state feature_gate_name="VolumeAttributesClass" >}}
 
 This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
-[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+[Volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
```

Comment:
(nit)
```suggestion
[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
```
Why? Because you can't `kubectl get volume`.